### DISCUSSION TITLE
Feature Request: Integration with Apple Shortcuts App

### DISCUSSION THREAD

**Discussion by johnbyer** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#734] [Ideas]
I'd love to see a future version of AlDente have integrations with Apple Shortcuts. The ability to set an AlDente charge limit based on user-defined actions would be spectacular for those who have hybrid offices, attend classes, etc. Being able to set calibration cycles based on user-defined circumstances would be excellent as well.

Keep up the great work!

**Comment by MatthiasKerbl** [comment]
Hi @johnbyer,

We have already started working on this and will slowly roll it out over the next couple of months.
#541 is the official thread for this.

Cheers,
Matthias

**Comment by MatthiasKerbl** [comment]
Hi @johnbyer,

Apple Shortcuts are now officially supported in AlDente Pro.

Cheers,
Matthias

**Comment by johnbyer** [comment]
Thanks so much for the update, that’s awesome, I’m stoked!

**Reply by MatthiasKerbl** [reply]
You're welcome! :)
You can read more about this feature on our blog here: https://apphousekitchen.com/feature-explanation-shortcuts/
